### ISSUE TITLE
Success Criteria have CSS Class for Guidelines in the 25 January 2023 Candidate Recommendation Draft

### ISSUE THREAD

**Issue by Grunet** [w3c/wcag#3100]
I was trying to update the scraping part of [WCAG of the Day](dwcag.org) to pull from the latest version of WCAG 2.2. (I think https://www.w3.org/TR/2023/CRD-WCAG22-20230125/) but I noticed the success criterion section HTML elements had a class of "guideline" set on them.

Previously they had a class of "sc" on them.

I know my script probably should have not been relying on these since they're not semantic, but it does seem a little odd now.

I will look into if a fix is possible, and failing that update my scraper to adjust to this

**Comment by Grunet** [comment]
The issue isn't present in the draft released on September 6 2022 https://www.w3.org/TR/2022/CR-WCAG22-20220906/

So presumably it was a change since then that led to it?

**Comment by Grunet** [comment]
The only PR that stands out to me as potentially relevant is #2768 for "Update build". I didn't see anything else that seemed potentially relevant since last September

**Comment by Grunet** [comment]
I have a feeling the generate-structure-xml.xslt file is involved, I just can't quite tell what I'm looking at yet

**Comment by Grunet** [comment]
Something must be overriding it because in the source HTML fragments the "sc" class is there https://github.com/w3c/wcag/blob/fff36a0898ad3b516e8666034af19dd3fed7dc61/guidelines/sc/20/abbreviations.html

**Comment by Grunet** [comment]
My best guess is it's this commit from the above PR that resulted in it https://github.com/w3c/wcag/commit/b450e3e884d20eb5e29f476ba7bc5a5db01735c2#diff-09513d2d9f91572ccfd60afda2fc523b90336f50017e1daa6eb5a5b775d6ace5

Since the "wcag:get-id" function (or something else? I still don't know what I'm looking at apologies) only seems to be used for when the success criteria HTML fragments are included, but not guidelines or principles - https://github.com/w3c/wcag/blob/0fbb2b7cb95ee34123b0dd620e8dc8e5128a98cd/xslt/generate-structure-xml.xslt#L95

**Comment by Grunet** [comment]
Let me know if there's anything more I can help with for this!

I'm still not really sure what I'm looking at but if there's a way I can help do let me know!

**Comment by rvantonisse** [comment]
Nice, I was into grabbing a list of success criteria when noticing this issue.

Poking @remibetin and @mhansma96, for possible effects on creating translations / data for the  [wai-wcag-em-report-tool](https://github.com/w3c/wai-wcag-em-report-tool/tree/main)

@Grunet Workaround for targetting success criteria only:

```javascript
document.querySelectorAll(".guideline > .guideline")
```
